### PR TITLE
UIORGS-234 Deleting a contact person assigned to multiple orgs creates UI error

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Purpose

https://issues.folio.org/browse/UIORGS-234

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: translations/ui-organizations/en.json [hunk 1]
```diff
@@ -221,6 +221,7 @@
   "contactPeople.addContact": "Add contact",
   "contactPeople.newContact": "New",
   "contactPeople.noFindContactPlugin": "no find-contact plugin",
+  "contactPeople.remove": "Removed contact",
```
Comment (aliaksei-chumakou) [inline translations/ui-organizations/en.json:224]: maybe it's better to make key corresponding to value, contactPeople.removedContact. what do you think?

Comment (NikitaSedyx) [reply translations/ui-organizations/en.json:224]: oh yeah, I wanted to use this, mb ctrl + z

**Comment by aliaksei-chumakou** [comment]
shouldn't it be resolved with soft-deleting records on the back-end?

**Review by aliaksei-chumakou — APPROVED** [review]

**Review by axelhunn — APPROVED** [review]

**Review by AndreiShumski2 — APPROVED** [review]

**Comment by NikitaSedyx** [comment]
do we have soft-delete on BE?

**Comment by NikitaSedyx** [comment]
and soft-delete doesn't help in this case, I think it will be another issue like "I removed contact, but it's available for another organization"

**Review by uladislausamets — APPROVED** [review]

**Pull request merged**